test/features/comments.feature

```diff
@@ -0,0 +1,89 @@
+@comments
+Feature: Comments
+
+    Scenario: The Add Comment form should not display for a non-logged-in user - instead they see a 'Login to comment' button
+        Given "Unathenticated" as the persona
+        Then I go to dataset "warandpeace"
```

Comment:
Then I go to dataset "<name>"

should really be a When, 

Given a persona should also have a logged in version and a not logged in version.

Reply:
Thanks.

Have updated to "When".

Not sure I understand the second part of your comment though - the test scenario after this one uses a logged in persona - is that what you mean?